### PR TITLE
Add "static_lambda" CS fixer rule

### PR DESCRIPTION
**Pull request by phansys**

### CODE REVIEW

**Comment by OskarStark** [comment]
Nice but this is only the config file for the dev-kit project itself, please modify the 

projects/.php_cs.dist.twig file too 

I am currently on a phone 📱, hope the path is correct 😌😎

File: .php_cs.dist [hunk 1]
```diff
@@ -41,7 +42,7 @@ $rules = [
     '@PHPUnit60Migration:risky' => true,
     '@PHP71Migration' => true,
     '@PHP71Migration:risky' => true,
-    'compact_nullable_typehint' => true,
```
Comment (OskarStark) [inline .php_cs.dist]: Please keep the order as is and only add your new rule

Comment (phansys) [reply .php_cs.dist]: Done.

**Comment by phansys** [comment]
Updated `project/.php_cs.dist.twig`.

**Comment by OskarStark** [comment]
@Soullivaneuh please merge an deploy

**Comment by OskarStark** [comment]
Thank you, but no Pull Requests were created, did you deploy the new code?

**Pull request merged**